src/app/pages/interest-form/interest-form.hbs

```diff
@@ -0,0 +1,203 @@
+            College Physics for AP® Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Concepts of Biology">
+            Concepts of Biology
+        </label>
+        <label>
+            <input id="adoption_interest_textbook_interests_" name="adoption_interest[textbook_interests][]" type="checkbox" value="Introduction to Sociology">
+            Introduction to Sociology
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Introduction to Sociology 2e">
+            Introduction to Sociology 2e
+        </label>
+        <label>
+            <input id="adoption_interest_textbook_interests_" name="adoption_interest[textbook_interests][]" type="checkbox" value="Introductory Statistics" class="columnizer-split">
+            Introductory Statistics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Prealgebra">
+            Prealgebra
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Precalculus">
+            Precalculus
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Economics">
+            Principles of Economics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Macroeconomics">
+            Principles of Macroeconomics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Macroeconomics for AP® Courses">
+            Principles of Macroeconomics for AP® Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Microeconomics">
+            Principles of Microeconomics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Principles of Microeconomics for AP® Courses">
+            Principles of Microeconomics for AP® Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Psychology">
+            Psychology
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="U.S. History">
+            U.S. History
+        </label>
+    </fieldset>
+    <fieldset class="labeled-inputs">
+        <label>
+            <span>Students per semester</span>
+            <input name="adoption_interest[students_per_semester]" type="number" required="true">
+        </label>
+    </fieldset>
+    <div class="section">
```

Comment:
This element appears to have no closing tag and doesn't seem to be necessary.  It can probably just be deleted.